.codeclimate.yml

```diff
@@ -13,6 +13,9 @@ plugins:
       - spec/
   reek:
     enabled: true
+    checks:
```

Comment:
As discussed before, the Reek README has a better suggestion which you might as well implement in full here